plugins/win-capture/game-capture.c

```diff
@@ -1595,7 +1625,7 @@ static void game_capture_tick(void *data, float seconds)
 			get_window_class(&gc->class, hwnd);
 
 			gc->priority = WINDOW_PRIORITY_CLASS;
-			gc->retry_time = 10.0f;
+			gc->retry_time = 10.0f*hook_rate_to_float(gc->config.hook_rate);
```

Comment:
Style:  The line length limit is 80 columns.  You'll have to line-break this line.